app/main.py

```diff
@@ -1,6 +1,21 @@
-from typing import Callable
+from typing import Callable, Any
+from functools import wraps
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_store = {}
+
+    @wraps(func)
+    def wrapper(*args: Any, **kwargs: Any) -> Any:
+        cache_key = (args, frozenset(kwargs.items()))
+
+        if cache_key in cache_store:
+            print('Getting from cache')
+            return cache_store[cache_key]
+        else:
```

Comment:
There is no need for `else` since the code will not execute after the `return` statement anyway. Please, remove the redundant `else`